doc/classes/AudioServer.xml

```diff
@@ -35,6 +35,26 @@
 				Generates an [AudioBusLayout] using the available buses and effects.
 			</description>
 		</method>
+		<method name="get_absolute_time" qualifiers="const" experimental="">
+			<return type="float" />
+			<description>
+				Returns the absolute time in seconds of the [AudioServer]'s timeline, based on the number of audio frames mixed. Used to schedule sounds to be played with higher precision timing, such as with [method AudioStreamPlayer.play_scheduled].
```

Comment:
"Higher" than what? Given the many things that could be said (`_process` method, physics ticks, the Timer node, the Time singleton, etc.), , it's probably not worth specifying. Not in this description, at least.
```suggestion
				Returns the absolute time in seconds of the [AudioServer]'s timeline, based on the number of audio frames mixed. Used to schedule sounds to be played with high precision timing, such as with [method AudioStreamPlayer.play_scheduled].
```